### ISSUE TITLE
update ruby version in tests

### ISSUE THREAD

**Issue by ceochronos** [cmckni3/rails-jasperserver#17]
Hi,

I was about to create an issue regarding the old version of `rest-client` but I see that issue has already been taken care of.

However, test are failing due to we are testing against an unsupported combination of Ruby and `rest-client`

Can we remove all unsupported ruby versions and add current versions? According to https://www.ruby-lang.org/en/downloads/branches/, as of today, the only supported versions are: 2.3, 2.4 y 2.5.

I think I can do it.

I will start working on it in a different branch in the fork I just created.

**Comment by cmckni3** [comment]
Yeah I haven’t updated this or tested it against a real report instance in awhile. #15 was to get a report instance running in Docker for development. I ran into licensing issues I believe.

The rails test application can be scrapped but keep the vcr recordings.

**Comment by ceochronos** [comment]
Testing is hard, I know; and as far as I know gem is working fine in rails 4.x and 5.x; I am using rails 5.1.6 in production.

I'll try to do something to make this work.

**Comment by cmckni3** [comment]
I’m fine with dropping support for Ruby 1.9 and bumping the major version.

**Comment by cmckni3** [comment]
I dropped support for Ruby < 2.3. Ruby 2.2 is still running on Travis but is allowed to fail.
